### PR TITLE
🦚 MVP Cleanup 2

### PR DESCRIPTION
**Pull request by SirajChokshi**
## Status:

:construction: In development

## Description

- Added mobile styles for PhotoGrid
- `LoadingIndicator` component 

## Todos

- [ ] More media queries
- [ ] Update `EntryDetail` layout

## Screenshots

https://user-images.githubusercontent.com/19193347/111911696-9359cb00-8a34-11eb-9c46-2181d1a836ee.mov

### CODE REVIEW

**Review by Qrtn — APPROVED** [review]
Looks good!! Love the loading indicator

**Review by laurenho025 — APPROVED** [review]
beautiful

File: src/components/ui/LoadingIndicator.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      }
+    }
+    @-webkit-keyframes spin {
+      from {
+        -webkit-transform: rotate(0deg);
+      }
+      to {
+        -webkit-transform: rotate(360deg);
+      }
+    }
```
Comment (Qrtn) [inline src/components/ui/LoadingIndicator.js:57]: Does this get taken care of webpack? Do you need to add it?

File: src/components/ui/LoadingIndicator.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    margin-right: ${(props) => (props.vertical ? 0 : '0.25em')};
+    --animation: spin 1.2s infinite ease-in-out;
+
+    @media (prefers-reduced-motion: reduce) {
+      --animation: none;
+    }
+
+    -webkit-animation: var(--animation);
+    -moz-animation: var(--animation);
+    -o-animation: var(--animation);
```
Comment (Qrtn) [inline src/components/ui/LoadingIndicator.js:39]: Same here.

File: src/components/ui/LoadingIndicator.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    animation: var(--animation);
```
Comment (Qrtn) [inline src/components/ui/LoadingIndicator.js:40]: If removing vendor prefixes, do we still need a CSS var?

**Pull request merged**